flows/engine/log.go

```diff
@@ -27,18 +27,19 @@ func (s *logEntry) Event() flows.Event   { return s.event }
 //------------------------------------------------------------------------------------------
 
 type logEntryEnvelope struct {
-	StepUUID   flows.StepUUID       `json:"step_uuid"   validate:"required"`
-	ActionUUID *flows.ActionUUID    `json:"action_uuid"`
-	Event      *utils.TypedEnvelope `json:"event"       validate:"required"`
+	StepUUID   flows.StepUUID       `json:"step_uuid,omitempty"`
```

Comment:
Making this optional so that we can error sessions in some cases when we can't resume then properly